src/main/routes/routes.ts

```diff
@@ -84,6 +84,12 @@ export default function(app: Application): void {
   app.post('/subscription-add', ensureAuthenticated, app.locals.container.cradle.subscriptionAddController.post);
   app.get('/case-event-glossary', app.locals.container.cradle.caseEventGlossaryController.get);
 
+  app.get('/subscription-case-search', app.locals.container.cradle.subscriptionCaseSearchController.get);
```

Comment:
You're missing ensureAuthenticated flag these pages are not accessible to public users

Reply:
flag has been added.